app/src/components/index.tsx

```diff
@@ -1,3 +1,11 @@
+import type {
+  FieldErrorProps,
+  InputProps,
+  LabelProps,
+} from "react-aria-components";
+// eslint-disable-next-line no-duplicate-imports
+import { FieldError, Input, Label } from "react-aria-components";
```

Comment:
Can you remove the eslint override by changing this to `export { FieldError, Input, Label } from 'react-aria-components'`

Reply:
Oh yeah that's much cleaner